charts/openmetadata/templates/deployment.yaml

```diff
@@ -73,8 +73,10 @@ spec:
             {{ .Values.livenessProbe | toYaml | indent 12 | trim }}
           readinessProbe:
             {{ .Values.readinessProbe | toYaml | indent 12 | trim }}
+{{- if (semverCompare ">=1.16-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Hello @naturalett . Any particular reason for this backward compatibility ?

We currently only support upto latest 3 major kubernetes versions as for the Kubernetes Versions Compatibility Matrix [here](https://endoflife.date/kubernetes).